autoload/neomake.vim

```diff
@@ -873,7 +875,9 @@ function! neomake#MakeHandler(job_id, data, event_type) abort
         if neomake#has_async_support()
             if !len(filter(copy(s:jobs), 'v:val.make_id == jobinfo.make_id'))
                 call neomake#utils#hook('NeomakeFinished', {
-                            \ 'file_mode': maker.file_mode})
+                            \ 'file_mode': maker.file_mode,
+                            \ 'bufnr': get(jobinfo, 'bufnr', -1),
```

Comment:
I am not sure whether `jobinfo.bufnr` always exists, as I am not familiar with the core internals of neomake (e.g. the anatomy of `jobinfo`). Could you please clarify to me?

In L855-L857, if `file_mode` is 0, `jobinfo.bufnr` is not used, which is the reason why I coded so. Please see also L693-695 where the original code reads `get(maker, 'bufnr', -1)`; it is about `maker.bufnr`.

However, it looks that `jobinfo.bufnr` is set whenever I tested, even if `file_mode` is 0. If you believe `jobinfo` always contains `bufnr`, I would fix by removing default as you suggested.